src/Executable.js

```diff
@@ -634,6 +634,14 @@ export default class Executable {
                     this._nodeAccountIds.index ===
                     this._nodeAccountIds.list.length - 1;
 
+                // Check if the request is a transaction receipt or record
+                // request to retry 10 times, because its getReceiptQuery/getRecordQuery
+                // are single node requests
+                if (isTransactionReceiptOrRecordRequest(request)) {
+                    await delayForAttempt(false, attempt, 500, 1000);
+                    continue;
```

Comment:
await delayForAttempt(
                        isLocalNode,
                        attempt,
                        this._minBackoff,
                        this._maxBackoff,
                    );